src/lib/utils.js

```diff
@@ -0,0 +1,111 @@
+//jshint esversion: 6
```

Comment:
Will you please add tests that cover the same functions moved to the new `utils` file?

Reply:
sure, working on that.